Building.md

```diff
@@ -0,0 +1,58 @@
+This is the general building guide for all FullScreenShenanigans projects. The guide will show you how to download and install any FullScreenShenanigans game as well as teach you about how to use the console to interact with the game.
+
+### Table of Contents
+
+1. [Obtaining the Source Code](#obtaining-the-source-code)
+2. [Build Process](#build-process)
+3. [Using the Console](#using-the-console)
+
+## Obtaining the Source Code
+
+All FullScreenShenanigans projects follow the following URL format: https://github.com/FullScreenShenanigans/project-name/ so, for example, if you want to download your own copy of FullScreenMario, you can get it at https://github.com/FullScreenShenanigans/FullScreenMario/. In order to play the game, you need to use one of the two following methods to obtain the source code:
+
+### Clone the project repository using GitHub:
+1. If you don't already have Git installed on your computer, go to [this page](http://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and follow the instructions to download Git.
+2. Once Git is downloaded, follow [these directions](https://help.github.com/articles/cloning-a-repository/) to clone the repository.
+3. See [Build Process](#build-process) to see how to build and play the game.
+
+### Download the latest release .zip of the project:
+1. Go to the GitHub page of your desired game.
+2. Click the `Download ZIP` button found on the right side of the page.
+3. Extract the .zip into a directory of your choice.
+4. Open Source/index.html in a browser (preferably Google Chrome).
+
+
+## Build Process
+
+FullScreenShenanigans projects use [Grunt](http://gruntjs.com/) to automate building, which requires you to [download Node.js](http://nodejs.org) first.
+
+Go to the root repository folder and run the following commands:
+
+    npm install
+    grunt
+    
+You may also have to manually install grunt first:
```

Comment:
I just did a bit of poking around, and you definitely have to install grunt globally first. Can you move the second code snippet onto the first line of the first code snippet, and remove its sentence?